docs/JavaScript_Basics/math.md

```diff
@@ -0,0 +1,11 @@
+Math is a built-in object that has properties and methods for mathematical constants and functions.
```

Comment:
@sneha122 Please add some styling as Headings should be big, imp points should be bold like this